custom_components/plugwise/config_flow.py

```diff
@@ -143,7 +143,7 @@ async def verify_connection(
         errors[CONF_BASE] = "response_error"
     except UnsupportedDeviceError:
         errors[CONF_BASE] = "unsupported"
-    except Exception:  # noqa: BLE001
```

Comment:
Nothing is complaining about broad exceptions anymore (or does it finally ingest it's last resort) 👍

Reply:
The `noqa` was the cause of a pylint-error, happy to remove it :smile: